packages/core/src/browser/quick-open/quick-open-service.ts

```diff
@@ -49,11 +49,13 @@ export namespace QuickOpenOptions {
 }
 
 @injectable()
-export class QuickOpenService {
+export abstract class QuickOpenService {
     /**
      * It should be implemented by an extension, e.g. by the monaco extension.
      */
     open(model: QuickOpenModel, options?: QuickOpenOptions): void {
         // no-op
     }
+
+    abstract refresh(): void;
```

Comment:
Some comment/documentation would be useful, to say what refresh does (or should do, if you're somebody implementing it).

Reply:
Done